windows/devices/uwb/UwbSession.cxx

```diff
@@ -295,3 +296,15 @@ UwbSession::DestroyImpl()
         throw e;
     }
 }
+
+std::vector<uint8_t>
+UwbSession::GetOobDataObjectImpl()
+{
+    auto applicationConfigurationParameters = GetApplicationConfigurationParameters({});
```

Comment:
Specifying none (empty vector) has a special meaning to request all parameters, right? If so, we should define a constant for this to make it more clear in the code. Probably makes sense to define this at/close/above the prototype for `UwbSession::GetApplicationConfigurationParameters()` in UwbSession.hxx. Eg. (example defines it here instead for simplicity):

```suggestion
    static constexpr std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> AllParameters = {};
    auto applicationConfigurationParameters = GetApplicationConfigurationParameters(AllParameters);
```

Reply:
Yes that's correct. I think the empty vector as a param is used somewhere else in the code (nocli maybe?), so it may make sense to define the constant at a broader level. I'll double check though